src/main/kotlin/uk/gov/dluhc/printapi/database/entity/PrintDetails.kt

```diff
@@ -16,20 +16,23 @@ data class PrintDetails(
     var applicationReference: String? = null,
     @get:DynamoDbSecondaryPartitionKey(indexNames = [SOURCE_TYPE_GSS_CODE_INDEX_NAME]) var sourceType: SourceType? = null,
     var vacNumber: String? = null,
+    var vacVersion: String? = "1",
```

Comment:
I'm not sure the logic for these defaults (`vacVersion`, `suggestedExpiry` , and possibly `certificateFormat`) live in the entity class, but they are fine here for now.

I could well imagine a scenario where the version of the certificate and the suggested expiry should be in `application.yml` so they can be "tuned"; and I can also imagine a scenario where we might want to print a certificate using template version that is not the default.
It feels like the service that is creating these entities via the repository should control these field values / know about the defaults .... but this is fine for now. We can (and probably will) refactor this as and when we have a story to set these values